### ISSUE TITLE
Clarify how to handle investment companies

### ISSUE THREAD

**Issue by Greenheart** [Klimatbyran/garbo#529]
These kinds of reports include many different numbers that often conflict with each others. For example if they have reported emissions for the companies they have invested in.

**Comment by elvbom** [comment]
Think this is still relevant to keep open, do you agree @eli6 @kaylawoodbury ?

**Comment by kaylawoodbury** [comment]
@elvbom Yeah, still relevant I'd say. This also applies to groups as well
